Directory.Build.props

```diff
@@ -6,7 +6,7 @@
         <ImplicitUsings>enable</ImplicitUsings>
         <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <WarningsNotAsErrors>NU1901;NU1902;NU1903;NU1904</WarningsNotAsErrors>
+        <WarningsNotAsErrors>NU1608;NU1901;NU1902;NU1903;NU1904</WarningsNotAsErrors>
```

Comment:
This worked for my local VS build but the build server is still failing.  I'll figure it out tomorrow.

Motivation: We need to use `System.Text.Json` 9.0.0 due to .net 9 targeting but the new Kiota dependencies want < 9.0.0 so there is a build warning that we can just ignore.

Reply:
Fix: [commit](https://github.com/BiblioNexusStudio/aquifer-server/pull/601/commits/5708803049260106a50e5402827afa43321464ec).  I _don't_ want the warnings totally hidden from the build output (via `NoWarn`) because they do deserve to eventually be fixed by updating the impacted NuGet packages but right now there's nothing we can do to fix them because we are on the latest versions.  This commit ensures that they still warn but that they don't fail the build.